backend/src/api/vulnerabilities.ts

```diff
@@ -203,9 +205,7 @@ class VulnerabilitySearch {
 
     if (groupBy) {
       const results = await qs.getRawMany();
-      // TODO: allow pagination of grouped-by results. For now, we just
-      // return one page max of grouped-by results.
-      return [results, results.length];
+      return [results, totalResults];
```

Comment:
thanks -- can you add a test in vulnerabilities.test.ts to ensure this grouped endpoint works and pagination works as well?